airbyte-integrations/connectors/destination-snowflake/build.gradle

```diff
@@ -56,7 +56,6 @@ dependencies {
     integrationTestJavaImplementation 'org.apache.commons:commons-lang3:3.11'
 
     implementation files(project(':airbyte-integrations:bases:base-java').airbyteDocker.outputs)
-    integrationTestJavaImplementation files(project(':airbyte-integrations:bases:base-normalization').airbyteDocker.outputs)
```

Comment:
Was this intentional, or the formatter being dumb

Reply:
Not intentional, formatter removed this. Let me put this back in. I recall you had something similar in the #connector-ops channel